extensions/wikia/AdMixExperiment/scripts/noVideo1.js

```diff
@@ -0,0 +1,116 @@
+require(['wikia.throttle'], function (throttle) {
+	$(function () {
+
+		var $adMixRecircWrapper;
+		var $footer = $('#WikiaFooter');
+		var $moduleBeforeRecirc;
+		var $rail = $('#WikiaRail');
+		var $recircWrapper;
+		var $window = $(window);
+		var adMixRecircWrapperHeight;
+		var bottomMargin = 20;
+		var breakpointSmall = 1023;
+		var globalNavigationHeight = $('#globalNavigation').outerHeight(true);
+		var moduleBeforeRecircHeight;
+		var viewportWidth = $(window).width();
+
+		function getStartPosition() {
+			var recircOffsetTop = parseInt($moduleBeforeRecirc.offset().top, 10) + moduleBeforeRecircHeight;
+			return recircOffsetTop - globalNavigationHeight;
+		}
+
+		function getStopPosition() {
+			var stopPoint = parseInt($footer.offset().top, 10);
+			adMixRecircWrapperHeight = $adMixRecircWrapper.outerHeight(true);
+			return stopPoint - globalNavigationHeight - adMixRecircWrapperHeight - bottomMargin;
+		}
+
+		function getMidPosition() {
+			// INCONTENT_BOXAD_1
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ACommentedCode)